geoportailv3/static/js/maincontroller.js

```diff
@@ -507,15 +536,19 @@ app.MainController.Lux3DManager = class extends ngeo.olcs.Manager {
 /**
  * @private
  * @param {string} cesiumURL The Cesium URL
+ * @param {angular.Scope} $rootScope The root scope.
  * @return {?app.MainController.Lux3DManager} The created manager.
  */
-app.MainController.prototype.createCesiumManager_ = function(cesiumURL) {
+app.MainController.prototype.createCesiumManager_ = function(cesiumURL, $rootScope) {
   if (!this.allow3d) {
     return null;
   }
   // [minx, miny, maxx, maxy]
   goog.asserts.assert(this.map_);
-  return new app.MainController.Lux3DManager(cesiumURL, this.map_, this.ngeoLocation_);
+  const cameraExtentInRadians = [5.31, 49.38, 6.64, 50.21].map(ol.math.toRadians);
+  const ol3dm = new app.MainController.Lux3DManager(cesiumURL, cameraExtentInRadians, this.map_, this.ngeoLocation_);
+  ol3dm.setRootScope($rootScope);
```

Comment:
What is this used for ?

Reply:
You are right, it is weird. Looking at it and fixing what can be.

Reply:
done in the constructor and other refactorings.